### PR TITLE
feat(mojaloop/#3445): nodejs upgrade

### PR DESCRIPTION
**Pull request by aaronreynoza**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,27 +1,49 @@
-FROM node:16.15.0-alpine as builder
+# Arguments
+ARG NODE_VERSION=lts-alpine
+
+# NOTE: Ensure you set NODE_VERSION Build Argument as follows...
+#
+#  export NODE_VERSION="$(cat .nvmrc)-alpine" \
+#  docker build \
+#    --build-arg NODE_VERSION=$NODE_VERSION \
+#    -t mojaloop/sdk-scheme-adapter:local \
```
Comment (oderayi) [inline Dockerfile:9]:
```suggestion
#    -t mojaloop/simulator:local \
```

File: Dockerfile [hunk 2]
```diff
@@ -1,27 +1,49 @@
...
+
 WORKDIR /opt/app
 
-# Create a non-root user: ml-user
-RUN adduser -D ml-user 
-USER ml-user
+# Create empty log file & link stdout to the application log file
+RUN mkdir ./logs && touch ./logs/combined.log
+# Links combined to stdout
+RUN ln -sf /dev/stdout ./logs/combined.log
```
Comment (oderayi) [inline Dockerfile:39]: Why is this necessary?

**Review by oderayi — CHANGES_REQUESTED** [review]
Looks good, just a question regrading the logs management.

File: Dockerfile (hunk 2 above)
Comment (aaronreynoza) [reply Dockerfile:39]: Not sure, it's added into every Dockerfile already, I'll investigate.

Comment (aaronreynoza) [reply Dockerfile:38]: By linking the standard output to the application log file, any output written to stdout by the application will be redirected and appended to the "combined.log" file. This allows us to access and monitor the application's log output.

Comment (oderayi) [reply Dockerfile:38]: Ok, thank you for the clarification. 
The `central-service-logger` also defaults to writing to that file when using the file transport for logging:
```
https://github.com/mojaloop/central-services-logger/blob/966826cab74713a2ba743fe1b89eb200acfb7c68/config/default.json#L10
```

**Review by oderayi — APPROVED** [review]
